src/AstroImages.jl

```diff
@@ -20,15 +20,17 @@ tuple with the data of each corresponding extension is returned.
 function FileIO.load(f::File{format"FITS"}, ext::Int=1)
     fits = FITS(f.filename)
     out = _load(fits, ext)
+    header = WCS.from_header(read_header(fits[ext], String))[1]
```

Comment:
Are you sure that `WCS.from_header(...)[1]` is always possible?  Consider that we have to deal with FITS files that may not have WCS information, too

Reply:
Yeah as I mentioned in the above comments, `WCS.jl` provides a default `WCSTransform` even for such files. We can detect that and use it accordingly.

Reply:
When I tried to run `WCS.from_header(read_header(fits[1], String))` on a FITS file without WCS information I got a 0-element array, so that accessing the first element threw an error.

Reply:
@giordano I can't say about the file you are testing but this seems to work just fine for me.
```julia
fname = tempname() * ".fits"
f = FITS(fname, "w")
indata = reshape(Float32[1:100;], 5, 20)
write(f, indata)
write(f, indata)
close(f)
size(WCS.from_header(read_header(FITS(fname)[1], String)))
```

Also, in this file that I have created I have only image data.

Reply:
That file doesn't have an empty header.

Reply:
Never mind, that file did have a header but the image was actually empty, in the sense that it has dimension 0, not really an interesting case for this application